classes/UpgradeContainer.php

```diff
@@ -685,6 +700,62 @@ public function getUpgradeConfigurationStorage(): UpgradeConfigurationStorage
         return new UpgradeConfigurationStorage($this->getProperty(self::WORKSPACE_PATH) . DIRECTORY_SEPARATOR);
     }
 
+    public function getDistributionApiService(): DistributionApiService
+    {
+        if (null !== $this->distributionApiService) {
+            return $this->distributionApiService;
+        }
+
+        $this->distributionApiService = new DistributionApiService();
+
+        return $this->distributionApiService;
+    }
```

Comment:
can you use this new getter in https://github.com/PrestaShop/autoupgrade/blob/97045a8b72d88f32c666a645ec0f6e9b2773d81a/classes/UpgradeContainer.php#L452